.github/workflows/main.yml

```diff
@@ -40,25 +40,18 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven
 
-      - name: Maven operation with Sonar
+      - name: Regular Build
+        run: |
+          ./mvnw -B -U clean verify
+
+      - name: Sonar Analysis
         if: matrix.sonar-enabled
         run: |
-          mvn -B -U -Pcoverage \
-          clean verify \
-          sonar:sonar \
+          ./mvnw -B -Pcoverage sonar:sonar \
           -Dsonar.projectKey=AxonFramework_extension-tracing \
           -Dsonar.organization=axonframework \
           -Dsonar.host.url=https://sonarcloud.io \
           -Dsonar.login=${{ secrets.SONAR_TOKEN }}
-        env:
```

Comment:
I noticed from failing builds on some extensions that the `GITHUB_TOKEN` is needed.
Check [this](https://github.com/AxonFramework/extension-mongo/runs/4532027476?check_suite_focus=true#step:6:141) for example =)

Reply:
Good point! Just added it.